src/frontend/components/LiveVideoPanel/index.spec.tsx

```diff
@@ -66,13 +89,90 @@ describe('<LiveVideoPanel />', () => {
     screen.getByText('application content');
   });
 
+  it('renders the correct viewers list if the user is not an instructor', () => {
+    mockGetDecodedJwt.mockReturnValue({
+      permissions: {
+        can_update: false,
+      },
+    } as DecodedJwt);
+
+    useParticipantsStore.setState({
+      participants: [
+        {
+          ...mockParticipant,
+          isInstructor: false,
+          isOnStage: false,
+        },
+      ],
+    });
+    useLivePanelState.setState({
+      currentItem: LivePanelItem.VIEWERS_LIST,
+      availableItems: [
+        LivePanelItem.APPLICATION,
+        LivePanelItem.CHAT,
+        LivePanelItem.VIEWERS_LIST,
+      ],
+    });
+
+    render(wrapInIntlProvider(<LiveVideoPanel video={mockVideo} />));
+
+    screen.getByRole('tablist');
+    screen.getByRole('tab', { name: 'application' });
+    screen.getByRole('tab', { name: 'chat' });
+    screen.getByRole('tab', { name: 'viewers' });
+
+    screen.getByText(mockParticipant.name);
+  });
+
+  it('renders the correct viewers list if the user is an instructor', () => {
+    mockGetDecodedJwt.mockReturnValue({
+      permissions: {
+        can_update: true,
+      },
+    } as DecodedJwt);
+
+    useParticipantsStore.setState({
+      participants: [
+        {
+          ...mockParticipant,
+          isInstructor: false,
+          isOnStage: false,
+        },
+      ],
+    });
+
+    useLivePanelState.setState({
+      currentItem: LivePanelItem.VIEWERS_LIST,
+      availableItems: [
+        LivePanelItem.APPLICATION,
+        LivePanelItem.CHAT,
+        LivePanelItem.VIEWERS_LIST,
+      ],
+    });
+
+    render(wrapInIntlProvider(<LiveVideoPanel video={mockVideo} />));
+
+    screen.getByRole('tablist');
+    screen.getByRole('tab', { name: 'application' });
+    screen.getByRole('tab', { name: 'chat' });
+    screen.getByRole('tab', { name: 'viewers' });
+
+    screen.getByText(mockParticipant.name);
```

Comment:
I don't see the difference with the previous test when the user is not an instructor. What should change ?